graql/language/rule-validation.feature

```diff
@@ -26,9 +26,13 @@ Feature: Graql Rule Validation
     Given graql define
       """
       define
-      person sub entity, plays employment:employee, owns name, owns email @key;
+      person sub entity,
+        plays employment:employee, plays scholarship:scholar,
+        owns name, owns nickname, owns email @key;
       employment sub relation, relates employee, owns start-date;
+      scholarship sub relation, relates scholar;
       name sub attribute, value string;
+      nickname sub name;
```

Comment:
```suggestion
      nickname sub attribute, value string;
```
In Grakn 2.0, the parent of a sub-attribute must be marked as `abstract`.

Given that none of the current tests actually make use of the type hierarchy, let's change the definition of `nickname`.